crates/api_crud/src/comment/delete.rs

```diff
@@ -65,26 +62,26 @@ impl PerformCrud for DeleteComment {
 
     let post_id = updated_comment.post_id;
     let post = Post::read(context.pool(), post_id).await?;
-    let recipient_ids = send_local_notifs(
-      vec![],
-      &updated_comment,
-      &local_user_view.person,
-      &post,
-      false,
-      context,
-    )
-    .await?;
+    let recipient_ids = context
+      .send_local_notifs(
+        vec![],
+        &updated_comment,
+        &local_user_view.person,
+        &post,
+        false,
+      )
+      .await?;
 
-    let res = send_comment_ws_message(
-      data.comment_id,
-      UserOperationCrud::DeleteComment,
-      websocket_id,
-      None, // TODO a comment delete might clear forms?
-      Some(local_user_view.person.id),
-      recipient_ids,
-      context,
-    )
-    .await?;
+    let res = context
+      .send_comment_ws_message(
+        &UserOperationCrud::DeleteComment.to_string(),
+        data.comment_id,
+        websocket_id,
+        None, // TODO a comment delete might clear forms?
```

Comment:
I dont think this comment makes sense. DeleteComment doesnt take any form id as parameter, so we also cant send one in the response.

Reply:
It was maybe a bug on the front end, I'll remove the comment.